### PR TITLE
Translate docs/contributing/index

### PR DESCRIPTION
**Pull request by mindsers**
The PR translate `docs/contributing/index`

### CODE REVIEW

File: docs/contributing/index.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
-title: Contributing to Gatsby.js
-description: Learn about contributing to one of the most welcoming communities helping develop the future of the web
+title: Contribuer à Gatsby.js
+description: Apprenez en plus sur la contribution à l'une des communauté les plus acceuillante en aidant à developper le future d'internet.
```
Comment (dantereve) [inline docs/contributing/index.md]:
```suggestion
description: Apprenez en plus sur la contribution à l'une des communautés les plus accueillantes en aidant à développer le futur d'internet.
```

File: docs/contributing/index.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 disableTableOfContents: true
 ---
 
-Thanks for being interested in contributing! We're so glad you want to help!
+Merci pour votre interet pour la contribution Nous somme heureux que vous souhaitiez nous aider !
```
Comment (dantereve) [inline docs/contributing/index.md]:
```suggestion
Merci de votre intérêt pour contribuer ! Nous sommes heureux que vous souhaitiez nous aider !
```

**Review by dantereve — CHANGES_REQUESTED** [review]
Great! I've spotted some spellings that have to be addressed before we can merge this in. 😉

File: docs/contributing/index.md (hunk 2 above)
Comment (mindsers) [reply docs/contributing/index.md]: I agree with everything except "pour contribuer". It sounds weird to me...

**Review by dantereve — APPROVED** [review]
👍

**Pull request merged**